pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1327,4 +1327,15 @@ self: super: {
   haskell-lsp_0_18_0_0 = super.haskell-lsp_0_18_0_0.override { haskell-lsp-types = self.haskell-lsp-types_0_18_0_0; };
   lsp-test_0_8_2_0 = (dontCheck super.lsp-test_0_8_2_0).override { haskell-lsp = self.haskell-lsp_0_18_0_0; };
 
+  snap-server = overrideCabal super.snap-server (drv: {
+    patches = [(pkgs.fetchpatch {
+      url = https://github.com/snapframework/snap-server/pull/126/commits/4338fe15d68e11e3c7fd0f9862f818864adc1d45.patch;
+      sha256 = "1nlw9lckm3flzkmhkzwc7zxhdh9ns33w8p8ds8nf574nqr5cr8bv";
+    })
+    (pkgs.fetchpatch {
+      url = https://github.com/snapframework/snap-server/pull/126/commits/410de2df123b1d56b3093720e9c6a1ad79fe9de6.patch;
+      sha256 = "08psvw0xny64q4bw1nwg01pkzh01ak542lw6k1ps7cdcwaxk0n94";
+    })];
```

Comment:
@jerith666 Thanks for this contribution.

Can you add a comment in the source code here explaining what these patches are, as well as when you expect they can be removed?

This will hopefully make it easier for the nixpkgs maintainers to figure out when this override is no longer needed.

Reply:
Wasn't sure if that was necessary given that the patch URLs contain the PR URL, but, done now!  :)

Reply:
When reviewing bumps to packages, it's a lot easier to just read the comments to see if they are relevant, rather than having to copy a link into a browser, skim, digest the material, then make a determination.

That being said, please use comments to describe what the patches are fixing